ops/Makefile

```diff
@@ -59,55 +63,103 @@ promote-%: .be-logged-in .valid-env-%
 
 wait-for-%-slot-commit: .valid-env-%
 	@echo "Starting wait for commit $(DEPLOYED_COMMIT) in $(SOURCE_SLOT) slot of $* at `date`"
-	until  [[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(SLOT_INFO_ENDPOINT) | jq -r .git.commit.id`" ]] ; do sleep 5; done;
+	if [[ "$*" == "prod" ]] ; then \
+	  until  [[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(PROD_SLOT_INFO_ENDPOINT) | jq -r .git.commit.id`" ]] ; do sleep 5; done; \
+	else \
+	  until  [[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(SLOT_INFO_ENDPOINT) | jq -r .git.commit.id`" ]] ; do sleep 5; done; \
```

Comment:
Question: Would it be possible to check for `prod` during the variable assignment where we form the endpoints? (lines 14-17 and 19-22) If so, I think can cut out these repetitive `if/else` checks.

Reply:
Noting here for the record: We discussed this in our DevOps standups this week, and have agreed that this isn't a showstopper for this PR. We may end up cleaning this up at a later date.